### DISCUSSION TITLE
Publish Pest stub

### DISCUSSION THREAD

**Discussion by legecha** [pestphp/pest#595] [Q&A]
Is it possible to publish the Pest test stubs? When I do `php artisan stub:publish` they don't show up.

If not, would this mean they also wouldn't load from the standard stubs folder (like standard published stubs would for instance)?

**Comment by owenvoke** [comment]
It looks like currently they just always will load from the Pest [stubs](https://github.com/pestphp/pest/tree/1.x/stubs) directory. As these stubs are used for `artisan pest:test` in the [`PestTestCommand`](https://github.com/pestphp/pest-plugin-laravel/blob/2.x/src/Commands/PestTestCommand.php#L65), we could probably just use a stubs path and publish these.

I'd be up for this as a feature, as I can see people wanting custom Pest stubs. Feel free to open a PR on the Laravel plugin! 👍🏻

**Reply by legecha** [reply]
Yeah I just realised this will be more for the laravel plugin side of things. Thanks for the info, I'll take a look at a PR :)
